src/types/account-membership-settings.d.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Account Membership Settings
+ * Description: You can use the Account Membership Settings endpoint to retrieve
+ * settings controlling Account Membership.
+ * DOCS: https://documentation.elasticpath.com/commerce-cloud/docs/api/advanced/settings/account-membership-settings/index.html
+ */
+import { Resource, RelationshipToOne, Identifiable } from './core'
+import {AccountAddress, AccountAddressEdit} from "./account-address";
```

Comment:
```suggestion
import { AccountAddress, AccountAddressEdit } from './account-address';
```
Use single quotes and spaces around imports.

Reply:
Can you please apply this change to the rest of the PR.